### PR TITLE
Docs/bugs rephrasing hacakathon

### PR DESCRIPTION
**Pull request by blankdots**
### Describe the pull request:

- [x] Bug fix
- [x] Documentation change

### Changes made:

changes based on this, leaving out new additions or big changes

https://docs.google.com/document/d/13bHYYNRbFYHO05WN7rrXjB7reiIHmM1_Mp5WRAEGPu0/edit

The focus was on typo fixing and rephrasing.

### CODE REVIEW

**Review by kjellp — APPROVED** [review]
LGTM, very nice round of clean up!

File: docs/connection.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-All Local EGA instances are connected to Central EGA using
+All `FederatedEGA` instances are connected to `CentralEGA` using
 [RabbitMQ](http://www.rabbitmq.com), a Message Broker, that allows the
 components to send and receive messages, which are queued, not lost, and
 resent on network failure or connection problems.
 
 The RabbitMQ message brokers of each SDA instance are the **only**
-components with the necessary credentials to connect to Central EGA
+components with the necessary credentials to connect to CentralEGA
```
Comment (aaperis) [inline docs/connection.md:10]:
```suggestion
components with the necessary credentials to connect to `CentralEGA`
```

File: docs/connection.md [hunk 2]
```diff
@@ -118,30 +118,30 @@ workflow to CentralEGA, using the following routing keys:
...
+instance, the latter can contact `CentralEGA` using the federated queue
 and the shovel mechanism in their local broker.
 
-CentralEGA should then see 2 incoming connections from that new LocalEGA
+`CentralEGA` should then see 2 incoming connections from that new `FederatedEGA`
 instance, on the given `vhost`.
 
 The exchanges and routing keys will be the same as all the other
-LocalEGA instances, since the clustering is done per `vhost`.
+FederatedEGA instances, since the clustering is done per `vhost`.
```
Comment (aaperis) [inline docs/connection.md:139]:
```suggestion
`FederatedEGA` instances, since the clustering is done per `vhost`.
```

File: docs/structure.md [hunk 3]
```diff
@@ -7,18 +7,18 @@ This section provides overview material for how the services can be deployed in
...
 
-2. The process of the Submitter annotating the archived data in an online portal at Central EGA, resulting in assigned accession numbers for items such as DataSet, Study, Files etc.
+2. The process of the Submitter annotating the archived data in an online portal at `CentralEGA`, resulting in assigned accession numbers for items such as DataSet, Study, Files etc.
 
 
-In a stand-alone setup, the deployed service has less remote synchronisation to worry about, but on the other hand need more components to also handle annotations/meta-data locally, as well as to deal with identifiers etc.
+In a stand-alone setup, the deployed service has less remote synchronisation to worry about, but on the other hand more components might be required (e.g ([orchestrator](services/orchestrator))) to also handle annotations/meta-data locally, as well as to deal with identifiers etc.
 
-The NeIC SDA is targeting both use cases in several projects in the Nordics.
+The NeIC SDA is targeting both types of setup, to open up the use of the re-use of components to more use cases than initially envisioned.
```
Comment (aaperis) [inline docs/structure.md:21]:
```suggestion
The NeIC SDA is targeting both types of setup but also to allow for the possibility to re-use components in more use cases than initially envisioned.
```

**Review by aaperis — APPROVED** [review]
minor corrections

**Pull request merged**